src/main/java/com/sprint/mission/discodeit/entity/MessageAttachmentId.java

```diff
@@ -0,0 +1,19 @@
+package com.sprint.mission.discodeit.entity;
+
+import java.io.Serializable;
+import java.util.UUID;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@EqualsAndHashCode
+public class MessageAttachmentId implements Serializable {
+
+  private UUID message;
+  private UUID attachmentId;
+
```

Comment:
```suggestion
  private UUID messageId;
  private UUID attachmentId;

```

attachmentId는 뒤에 Id가 붙어 있는데 message엔 없네요?
일관성 있게 messageId로 하면 어때요?